helm/dagger/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and is generated by [Changie](https://github.com/miniscruff/changie).
 
 
 
+## v0.12.7 - 2024-09-02
+
+
```

Comment:
We should not have releases with no changes. The change worth mentioning here is that the Engine version now defaults to `v0.m.p`, same as we do for SDKs that may not have any changes.

Reply:
Resolved, will update the release guide for next time.